src/main/java/com/squareup/javapoet/CodeBlock.java

```diff
@@ -269,6 +278,13 @@ public Builder addStatement(String format, Object... args) {
       return this;
     }
 
+    public Builder addStatement(CodeBlock codeBlock) {
```

Comment:
The thing is I create a code for statement through a few iterations, so I first use a codeblock for a few times and then need to wrap it up as a statement. Maybe I've missed a method to do this?